src/pure/pure.mli

```diff
@@ -62,9 +63,10 @@ val initial_state : file_path -> state
     [Cmd_Error] constuctor). *)
 val handle_command : state -> Command.t -> command_result
 
-(** [handle_tactic st tac] evaluated the tactic [tac] in state [st], returning
-    a new proof state (with [Tac_OK]) or an error (with [Tac_Error]). *)
-val handle_tactic : proof_state -> Tactic.t -> tactic_result
+(** [handle_tactic st tac] evaluates the tactic [tac] in state [st], for a
+    symbol or a definition with exposition [expo] returning a new proof state
+    (with [Tac_OK]) or an error (with [Tac_Error]). *)
+val handle_tactic : proof_state -> Terms.expo -> Tactic.t -> tactic_result
```

Comment:
Documentation needs update here, what is `Terms.expo`?

Reply:
I mean the beggining of the doc comment sorry, sorry, the expo parameter is missing.

Reply:
Yeah it seems that expo should be part of the state.

Reply:
Fixed in https://github.com/Deducteam/lambdapi/pull/422/commits/06a69ca0c614573effeb0eeaa5dbb0938f7ca56a